src/engraving/iengravingconfiguration.h

```diff
@@ -84,8 +84,8 @@ class IEngravingConfiguration : MODULE_EXPORT_INTERFACE
     virtual Color unlinkedColor() const = 0;
     virtual muse::async::Channel<Color> unlinkedColorChanged() const = 0;
 
-    virtual Color frameColor() const = 0;
-    virtual muse::async::Channel<Color> frameColorChanged() const = 0;
+    virtual Color borderColor() const = 0;
+    virtual muse::async::Channel<Color> borderColorChanged() const = 0;
```

Comment:
This method is also about something different (also about those _elements_ called "frame") so should not be renamed

Same for everything in engravingconfiguration.h/cpp